src/components/signin/Signin.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { Box, Button } from '@mui/material';
+import styled from 'styled-components';
+
+export const FlexColumn = styled(Box)({
```

Comment:
this can be just use div, not need to complicate the code if it is not needed. It will influence the app behaviour in future